bucket/rakudo-star.json

```diff
@@ -13,7 +13,9 @@
     "bin": [
         "bin\\perl6.exe",
         "bin\\raku.exe",
-        "bin\\rakudo.exe"
+        "bin\\rakudo.exe",
+        "share\\perl6\\site\\bin\\zef",
```

Comment:
This line will shim to the same name as the next line. One will need to be renamed. Is this a perl script without an extenstion?

Reply:
yes, it's a PERL6 script.
I guess I'll remove the zef.bat. Would that be OK for you?

Reply:
Ok, tested now. If only one of those is allowed then if has to be zef.bat. Both Powershell and CMD work only with zef.bat.
CMD doesn't like the plain zef :|